### ISSUE TITLE
Open to contributors

### ISSUE THREAD

**Issue by alexlouden** [SublimeText/Terraform#36]
I haven't been using Terraform for a while, and I've mostly moved to using VS Code so I haven't been able to give this much attention. (The [TF extension for VS Code](https://github.com/mauve/vscode-terraform) looks really good too!)

From now on I'll just merge and release PRs when I can, if anyone would like to help out then I'm happy to add you as a contributor!

**Comment by notnmeyer** [comment]
I'd be interested in helping out, mainly to get an updated package out :)

**Comment by jeffbyrnes** [comment]
I’d give a hand as well, I still use Terraform & Sublime Text daily.

**Comment by FichteFoll** [comment]
As I mentioned in https://github.com/alexlouden/Terraform.tmLanguage/pull/58, we (the ST community) would like to adopt this package into the [SublimeText](https://github.com/SublimeText/) GitHub org, where we collect various abandoned packages so that they can keep being maintained by motivated individuals that are still in touch with ST.

If you would like to go ahead with this proposal, feel free to transfer the repo to me and I will manage the rest (moving it to the org, setting up permissions, redirecting the Package Control entry) as I am an admin of that org.

**Comment by alexlouden** [comment]
Thanks @FichteFoll - that sounds like a great initiative. I'm happy to transfer it, if that's okay with @radeksimko? 

Also cc @patrickrgaffney - you've contributed the most to this project! 🙏 Any thoughts?

**Comment by radeksimko** [comment]
That's okay with me 👍🏻 

I didn't really have much time to contribute more lately. Also the HashiCorp Editor Experience team continues to manage the TextMate grammar at https://github.com/hashicorp/syntax which is now used by GitHub Linguist (as opposed to this repo) to highlight HCL and Terraform on GitHub.

Which is to say you could reduce the scope of this repo to just Sublime grammar and/or reuse that maintained TM grammar if you want.

**Comment by alexlouden** [comment]
Great, thanks @radeksimko. I've initiated the transfer, over to you @FichteFoll! 🫡

**Comment by FichteFoll** [comment]
Thank you for your trust. I have transferred the repository to https://github.com/SublimeText/Terraform and maintained the previous contributors (or rather, Github did so by default).

I have also filed a PR at the Package Control channel to update the location (though it should follow GitHub's redirects by default): https://github.com/wbond/package_control_channel/pull/8954

I'll set aside some time over the weekend to merge my open PR, submit a few other changes I have already done locally and also clean up the package a bit, removing the legacy stuff and turning it into something for ST4+ only.
